TheMovieDb/Models/MovieDetail.swift

```diff
@@ -0,0 +1,24 @@
+//
+//  MovieDetail.swift
+//  TheMovieDb
+//
+//  Created by Alex on 11/18/21.
+//
+
+import Foundation
+
+struct MovieDetail: Codable {
+    let poster_path: String?
+    let overview: String
+    let backdrop_path: String?
+    var items: [String] = ["", "", ""]
```

Comment:
What is the purpose of this array?

Reply:
To store the poster path, overview and backdrop path in one array. And that way a have a way to centralize the movie details value.

Reply:
But are we using it in the app? I couldn't find where are we using this property.

Reply:
The app creates an instance of this object, but I've removed the backdrop_path since I'm only showing the poster and movie description.